resources/fetchers/file_system_fetcher_test.go

```diff
@@ -39,18 +42,27 @@ func TestFileFetcherFetchASingleFile(t *testing.T) {
 		Patterns: filePaths,
 	}
 
+	osUserMock := &utils.MockOSUser{}
+	osUserMock.EXPECT().GetUserNameFromID(mock.Anything, mock.Anything).Return("root", nil)
```

Comment:
Relevant to all of the mock instances

Reply:
I think that we can safely assume that our code will never run on windows.
So can we use this one ?

Can we use the 
```
//go:build !windows
// +build !windows
```

Like in the following example? 
https://github.com/elastic/beats/pull/30808/files#diff-304eb22795a7b56d19543ff398610f0845f05e42eb606ddd8953545521fcea4e


It's not a must, I wonder if we use mock without asserting on input, the test will be less robust.
I mean, regardless of the input it will always return the same value.

I'll approve it regardless

Reply:
Thank you @ofiriro3.
It's not only windows, the `staff` group which is relevant here, might vary among different UNIX systems.